<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeBookGenericWidget</class>
 <widget class="QWidget" name="RecipeBookGenericWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="add_to_meal_plan">
     <property name="text">
      <string>Add to Meal Plan</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Recipes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Current Recipe</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="recipes_list"/>
   </item>
   <item row="2" column="1">
    <widget class="QTextBrowser" name="recipe_view"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="remove_recipe">
     <property name="text">
      <string>Remove Recipe From Book</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="delete_book">
     <property name="text">
      <string>Delete Book</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
